b64json/b64json.go

```diff
@@ -13,7 +42,20 @@ func Encode(v interface{}) (*string, error) {
 		return nil, err
 	}
 
-	b64Token := base64.StdEncoding.EncodeToString(jsonToken)
+	// Struct fields are marshalled in order of declaration, but we can't easily change the order
+	// OrderedMap fields are always marshalled in order, so we bounce through it
+	o := orderedmap.New()
+	err = o.UnmarshalJSON(jsonToken)
+	if err != nil {
+		return nil, err
+	}
+	o.Sort(jsonSort)
+	orderedJson, err2 := json.Marshal(o)
```

Comment:
can't reuse `err`?

Reply:
The := syntax wouldn't let me shadow err, would you like a predeclaration of orderedJson, rather?